yolapy/models/subscription.py

```diff
@@ -5,54 +5,42 @@ class Subscription(object):
 
     """Construct a Subscription.
 
-    The data in a Subscription instance does not persist until it has
-    been saved.
-
-    :param auto_renew: Bool
-    :param billing_date: Datetime
-    :param deprovision_date: Datetime
-    :param expiry_date: Datetime
-    :param start_date: Datetime
+    :param id: Str, subscription's 32 char ID
+    :param created_at: Str, datetime when subscription was created
```

Comment:
So, you're suggesting to remove description where it's obvious?

It's a wrapper for data returned from service - since we don't do any format conversion, we can just describe here what is returned.